lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/context/EvaluatorContext.java

```diff
@@ -170,6 +170,26 @@ public synchronized void submitContext(final Configuration contextConfiguration)
     this.contextControlHandler.send(contextControlProto);
   }
 
+  public synchronized void submitContext(final String contextConf) {
```

Comment:
That does not work, because the request is coming from the .NET side, not from the Java side.